flink-table-store-spark/src/test/java/org/apache/flink/table/store/spark/SparkReadITCase.java

```diff
@@ -281,6 +281,176 @@ public void testAddColumn() throws Exception {
         assertThat(results.toString()).isEqualTo("[[8]]");
     }
 
+    @Test
+    public void testRenameColumn() throws Exception {
```

Comment:
Add these tests:
1. Table can still be read after column names are changed. See `testAddColumn` for how to create such test.
2. Swap two column names
    ```sql
    ALTER TABLE T RENAME COLUMN a to tmp;
    ALTER TABLE T RENAME COLUMN b to a;
    ALTER TABLE T RENAME COLUMN tmp to b;
    ```
    And check that table can still be read and order of columns is not disturbed.